backend/src/paymaster/index.ts

```diff
@@ -55,7 +54,7 @@ export class Paymaster {
 
       return returnValue;
     } catch (err) {
-      throw new Error('Transaction Execution reverted')
+      throw new Error('Failed to process request to bundler')
```

Comment:
Could this not also carry the error message and a hint to try again or contact support?